news/_posts/releases/2018-10-10-prestashop-1-6-1-22-maintenance-release.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title:  "Release of PrestaShop 1.6.1.22"
+subtitle: "Maintenance version of the 1.6.1.x branch"
+date:   2018-10-10 18:00:00
+authors: [ AntoineThomas ]
+icon: icon-leaf
+tags:
+ - version
+ - maintenance
+ - patch
+ - releases
+ - 1.6.1.x
+ - 1.6
+---
+
+PrestaShop 1.6.1.22 is now available. It fixes an important bug introduced in 1.6.1.21:
+
+- A patch has been backported from 1.7.x in 1.6.1.21, but part of the code was missing and it can block a shop. Shops in 1.6.1.21 must upgrade to 1.6.1.22 as soon as possible.
```

Comment:
"A backported patch from 1.7.x was missing a line of code. This issue is not noticeable right away, but it can block a shop after a short number of days."